### DISCUSSION TITLE
Enable a Connector to communicate with multiple DAPS (Dataspaces)

### DISCUSSION THREAD

**Discussion by wjost** [International-Data-Spaces-Association/IDS-Messaging-Services#258] [Q&A] [answered]
Currently a Connector can only register to one DAPS, hence only be participant at one dataspace. However it would be essential to be able to participate with one connector multiple dataspaces, hence it is necessary to be able to pass the associated DAPS token and/or the address of the associated DAPS with the corresponding request.

**Comment by ghost** [comment]
Hello and thank you for the feedback, we will take a look!

**Comment by ghost** [comment]
After an initial estimate of the changes needed, this idea will be forwarded internally to the respective architecture groups, as multiple components and concepts would need to be addressed than just the IDS-Messaging-Services to implement this feature request.

**Comment by HeinrichPet** [comment, marked as answer]
IDSA internal ideas of having multiple DAPS: https://industrialdataspace.jiveon.com/docs/DOC-2832 (depricated) see new discussion document here: https://github.com/International-Data-Spaces-Association/IDS-ThinkTank/tree/main/ids-daps-federation

**Comment by ghost** [comment]
Will move it to the discussion section, as this requires conceptual changes that we can't address only here in the context of this repo. (Are there then multiple connector certificates from different DAPS etc.?).

**Comment by HeinrichPet** [comment]
DSA internal ideas of having multiple DAPS: https://github.com/International-Data-Spaces-Association/IDS-ThinkTank/tree/main/ids-daps-federation
